test/LinkedCellsTest.cpp

```diff
@@ -1,219 +1,218 @@
+//
+//  outflow_pc.linear([](Particle &p) {
+//    p.position = {0.5, -0.5, 0.5};
+//  });
+//
+//  outflow_pc.boundary([](auto p) {});
+//  outflow_pc.refresh();
+//
+//  auto counter = 0;
+//  outflow_pc.linear([&counter](auto &p) {
+//    counter++;
+//  });
+//  EXPECT_EQ(counter, 0);
+//}
+//
+//TEST_F(LinkedCellsTest, test_outflow_front) {
+//
+//  outflow_pc.linear([](Particle &p) {
+//    p.position = {0.5, 0.5, 1.5};
+//  });
+//
+//  outflow_pc.boundary([](auto p) {});
+//  outflow_pc.refresh();
+//
+//  auto counter = 0;
+//  outflow_pc.linear([&counter](auto &p) {
+//    counter++;
+//  });
+//  EXPECT_EQ(counter, 0);
+//}
+//
+//TEST_F(LinkedCellsTest, test_outflow_back) {
+//
+//  outflow_pc.linear([](Particle &p) {
+//    p.position = {0.5, 0.5, -1.5};
+//  });
+//
+//  outflow_pc.boundary([](auto p) {});
+//  outflow_pc.refresh();
+//
+//  auto counter = 0;
+//  outflow_pc.linear([&counter](auto &p) {
+//    counter++;
+//  });
+//  EXPECT_EQ(counter, 0);
+//}
+//
+///*#### REFLECTING CONDITIONS #####*/
+//
+//TEST_F(LinkedCellsTest, test_reflecting_left) {
+//
+//  reflecting_pc.linear([](Particle &p) {
+//    p.position = {0.25, 0.5, 0.5};
+//  });
+//
+//  reflecting_pc.boundary([](std::tuple<Particle &, Particle &> p) {
+//    auto [p1, p2] = p;
+//    EXPECT_EQ(p2.position[0], -0.25);
+//  });
+//  reflecting_pc.refresh();
+//}
+//
+//TEST_F(LinkedCellsTest, test_reflecting_right) {
+//
+//  reflecting_pc.linear([](Particle &p) {
+//    p.position = {0.75, 0.5, 0.5};
+//  });
+//
+//  reflecting_pc.boundary([](std::tuple<Particle &, Particle &> p) {
+//    auto [p1, p2] = p;
+//    EXPECT_EQ(p2.position[0], 1.25);
+//  });
+//}
+//
+//TEST_F(LinkedCellsTest, test_reflecting_bottom) {
+//
+//  reflecting_pc.linear([](Particle &p) {
+//    p.position = {0.5, 0.25, 0.5};
+//  });
+//
+//  reflecting_pc.boundary([](std::tuple<Particle &, Particle &> p) {
+//    auto [p1, p2] = p;
+//    EXPECT_EQ(p2.position[1], -0.25);
+//  });
+//  reflecting_pc.refresh();
+//}
+//
+//TEST_F(LinkedCellsTest, test_reflecting_top) {
+//
+//  reflecting_pc.linear([](Particle &p) {
+//    p.position = {0.5, 0.75, 0.5};
+//  });
+//
+//  reflecting_pc.boundary([](std::tuple<Particle &, Particle &> p) {
+//    auto [p1, p2] = p;
+//    EXPECT_EQ(p2.position[1], 1.25);
+//  });
+//}
+//
+//TEST_F(LinkedCellsTest, test_reflecting_back) {
+//
+//  reflecting_pc.linear([](Particle &p) {
+//    p.position = {0.5, 0.5, 0.25};
+//  });
+//
+//  reflecting_pc.boundary([](std::tuple<Particle &, Particle &> p) {
+//    auto [p1, p2] = p;
+//    EXPECT_EQ(p2.position[2], -0.25);
+//  });
+//  reflecting_pc.refresh();
+//}
+//
+//TEST_F(LinkedCellsTest, test_reflecting_front) {
+//
+//  reflecting_pc.linear([](Particle &p) {
+//    p.position = {0.5, 0.5, 0.75};
+//  });
+//
+//  reflecting_pc.boundary([](std::tuple<Particle &, Particle &> p) {
+//    auto [p1, p2] = p;
+//    EXPECT_EQ(p2.position[2], 1.25);
+//  });
+//}
```

Comment:
Why is this whole file commented out? Are the tests moved? Fix or delete, avoid keeping dead code.